lms/djangoapps/lms_xblock/mixin.py

```diff
@@ -26,6 +27,7 @@ def to_json(self, access_dict):
             return {unicode(k): access_dict[k] for k in access_dict}
 
 
+@XBlock.wants('partitions')
```

Comment:
@nasthagiri This is the mixin, which needs to reference the runtime service (no direct reference to course here).